.github/workflows/triage.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Triage"
+
+on: # yamllint disable-line rule:truthy
+  pull_request: null
+
+jobs:
+  label:
+    name: "Label"
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Add labels based on branch name"
+        if: "github.event.action == 'opened'"
+        uses: "actions/github-script@0.8.0"
+        with:
+          github-token: "${{ secrets.ERGEBNIS_BOT_TOKEN }}"
+          script: |
+            const branchPrefixLabels = {
+              enhancement: "feature",
```

Comment:
Not sure yet, but I am thinking about starting to use

- label names that match branch prefixes
- branch prefixes that match types of changes laid out in https://keepachangelog.com/en/1.0.0/

That is

- `add`
- `change`
- `deprecate`
- `fix`
- `remove`
- `security`